Tests.Integration.Presentation.Web/ItSystem/V2/ItSystemsInternalApiV2Test.cs

```diff
@@ -238,6 +238,22 @@ public async Task GET_Many_Internal_As_StakeHolder_With_ChangesSince_Filter()
             Assert.Equal(new[] { system3.Uuid, system1.Uuid }, dtos.Select(x => x.Uuid).ToArray());
         }
 
+        [Fact]
+        public async Task GET_Many_Internal_As_StakeHolder_With_NameEqual_Filter()
+        {
+            //Arrange
+            var (cookie, organization) = await CreateCookieStakeHolderUserInNewOrganizationAsync();
+            await ItSystemHelper.CreateItSystemInOrganizationAsync(CreateName(), organization.Id, AccessModifier.Public);
+            var system2 = await ItSystemHelper.CreateItSystemInOrganizationAsync(CreateName(), organization.Id, AccessModifier.Public);
```

Comment:
make sure the first one also contains the name of the second as part of it's name.. same comment as for the interface test